spec/features/guest_user/views/visitor_can_visit_categories_show_spec.rb

```diff
@@ -23,7 +23,7 @@
     it "and I see an error message" do
       create(:category, title: "Magic")
 
-      expect { visit '/magic' }.to raise_error(ActionView::Template::Error)
+      expect { visit '/magic' }.to raise_error(ActionController::RoutingError)
```

Comment:
ActionView::Template::Error -> ActionController::RoutingError

Why tho?

Reply:
This route resolves to stores#show, and since this isn't a valid store, the error is a routing error as defined in the ApplicationController not_found method.

Reply:
```
def not_found
      raise ActionController::RoutingError.new('Not Found')
end
```